<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="convert_ground" pkg="safer_stack" type="tf_pointcloud.py" output="screen">
        <remap from="input" to="/rtabmap/cloud_ground"/>
        <remap from="output" to="/rtabmap/cloud_ground_transformed"/>
    </node>
</launch>